app/models/user.rb

```diff
@@ -52,6 +52,15 @@ def switch_enabled
       save
     end
 
+    def toggle_role
+      if self.role == "default"
+        self.role = "merchant"
+      elsif self.role == "merchant"
+        self.role = "default"
+      end 
+      save
+    end 
+
```

Comment:
What I would do here nico:

```
self.role = self.role == "default" ? "merchant" : "default"
```

This is called a _ternary_ operator. `self.role` will be assigned the first one ("merchant") if the condition evaluates to true and the second one if false.

Reply:
`self.role == "default" ?`
That is the condition part

Reply:
btw I'm 2 branches ahead so I'm just gonna add this change on my most recent branch

Reply:
btw I'm 2 branches ahead so I'm just gonna add this change on my most recent branch